roles/forkliftcontroller/templates/deployment-validation.yml.j2

```diff
@@ -48,11 +48,20 @@ spec:
               value: 'false'
```

Comment:
@pemcg We need to ensure the configmap exists before we can setup volumes/mounts , otherwise it will create a conflict and the deployment will not be able to bring up the validation pod because some volumes are missing. The tasks/main.yml is the place for this , see here : https://github.com/konveyor/forklift-operator/blob/main/roles/forkliftcontroller/tasks/main.yml#L77